socialmedia.md

```diff
@@ -1,20 +1,118 @@
+* Audience: Anyone interested in your content or theme, i.e. education, school, teaching, children.
+* Recommended Use: To share videos that promote your school and learning environment. Christmas videos, shows, assemblies.
+
+
+### Our Social Media Accounts
+
+The school will have a presence on the following social networks: __Facebook, Twitter, LinkedIn, Instagram, Pinterest, Snapchat and Youtube__. Where possible, the account will be listed as a business or company under the school’s name, rather than as a personal or individual. It is important for us, as a school, to maintain a presence on these platforms to discourage others from creating a derogatory account in our name.
+
+The school will actively post on the following social networks: __Facebook, Twitter and Instagram.__
+Employees at the school are permitted to have personal accounts on any social networks they choose, providing they follow the rules outlines in Part 2 of this document and do not attempt to impersonate the school through an unofficial account.
+
+#### Links to Our Accounts
+* Facebook: __Add Here__
+* Twitter: __Add Here__
+* Instagram: __Add Here__
+* LinkedIn: __Add Here__
+* Snapchat: __Add Here__
+* Pinterest: __Add Here__
+* Youtube: __Add Here__
+
+### General Rules and Restrictions
+
+#### Posting on Social Media
+* Content will be posted to official school pages only.
+* There will be no content posted to a private page or closed group.
+* All posts are considered public, regardless of where they are posted, and should be treated as such.
+* The original post must be posted on a school page or account, but can then be shared by others from this place.
+* Any photographs or videos taken within school grounds, or during school time, should only be shared to a school account. 
+
+#### Photographs and Video
+* Media will be posted to school accounts only.
+* If the school keep a list of children who do not have parental consent to be online, this should be cross referenced before every post (and an audit trail of this kept for future reference).
+*	All children should be made aware a photo/video is being taken.
+*	We have our own “photo word” which must be said at all times before a photo is taken. This word is iconic to our school and is used to ensure all children know a photo is going to be taken. Our photo word is __“Swan”__.
+*	Children are always shown the photo after it is taken.
+*	There will be no media uploaded which contains one single child.
+*	There will be no media uploaded which contains identifiable features, such as name badges, name stickers on books, and so on.
+*	Employees will not post anything which may bring the school into disrepute or portray the school in a negative light.
+
+#### Engaging with Parents and Pupils
+*	Employees will not engage with parents or pupils via a closed or private account.
+*	All interactions must happen publicly and through school official channels only.
```

Comment:
I am not sure a parent complaining about how bad a school is should be public, maybe adjust to All interactions must happen though the official school accounts, all interactions are encourage to be public where feasible possible and suitable for that to happen.